tests/datastore_postgresql/conftest.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from testing_support.fixtures import (code_coverage_fixture,
+        collector_agent_registration_fixture, collector_available_fixture)
+
+_coverage_source = [
+    'newrelic.hooks.database_psycopg2',
```

Comment:
```suggestion
    'newrelic.hooks.database_postgresql',
```

Should this not be postgresql?